Makefile

```diff
@@ -66,27 +69,15 @@ test:
 bench:
 	go test -timeout=30s -bench=. $$(go list ./...)
 
-check: check-fmt ineffassign lint staticcheck vet
-
-check-fmt:
-	./contrib/scripts/check-fmt.sh
-
-ineffassign:
-	./tools/ineffassign .
-
-lint:
-	./tools/golint -set_exit_status $$(go list ./...)
-
-# Ignored staticcheck warnings:
-# - SA1019 deprecation warnings: https://staticcheck.io/docs/checks#SA1019
-# - ST1000 missing package comment: https://staticcheck.io/docs/checks#ST1000
-staticcheck:
-	./tools/staticcheck -checks="all,-SA1019,-ST1000" ./...
-
-vet:
-	go vet $$(go list ./...)
+ifneq (,$(findstring $(GOLANGCILINT_WANT_VERSION),$(GOLANGCILINT_VERSION)))
```

Comment:
should we use some sort of semver comp here, i.e. `1.40.2` should be ok too, right? Or do we specifically want the exact match

Reply:
Sure, I can drop `PATCH` from the version check.